docs/20_Configuration.md

```diff
@@ -22,22 +22,30 @@ However, you will find a link that will redirect you to **MongoDB CRUD** dedicat
+| LOG_LEVEL                     | String  | Optional | `info`                                    | Level of the log. It can be one of the following: `trace`, `debug`, `info`, `warn`, `error`, `fatal`.                                                                          |
+| EXPOSE_METRICS                | Boolean | Optional | `false`                                   | Specifies if Prometheus metrics should be exposed or not.                                                                                                                      |
 | ALLOW_DISK_USE_IN_QUERIES     | Boolean | Optional | -                                         | Sets the `allowDiskUse` option in the MongoDB queries. It is useful when working with MongoDB Views requiring heavy aggregations (added in v6.0.2, works with MongoDB >= 4.4). |
 
 :::warning
-Using `ALLOW_DISK_USE_IN_QUERIES` (either with `true` or `false` values) with a MongoDB version lower than 4.4 will make all the GET calls unusable, since the MongoDB cluster will raise an error for the unrecognized option `allowDiskUse`.
+Using `ALLOW_DISK_USE_IN_QUERIES` (either with `true` or `false` values) with a MongoDB version lower than 4.4 will make all the GET calls unusable, since the MongoDB cluster will raise an error for the unrecognized option `allowDiskUse`.
+
+It is also important to notice that starting from MongoDB v6.0 new server property [`allowDiskUseByDefault`](https://www.mongodb.com/docs/manual/reference/parameters/#mongodb-parameter-param.allowDiskUseByDefault)
+is introduced with its value set to `true` by default. Consequently, the default behaviour for that MongoDB version
+is that for queries with pipeline stages using more than 100 MB of memory to execute, the database automatically
+write temporary files on disk to support those queries.  
+This default behavior can be disabled for _all_ queries by setting `ALLOW_DISK_USE_IN_QUERIES` to `false`.  
 :::
 
-In case you want to use [Client Side Field Level Encryption](https://docs.mongodb.com/manual/core/security-client-side-encryption/), you should also include several specific Environment Variables, either you are using the [encryption with Google Platform Cloud](./30_Encryption_configuration.md#configure-csfle-with-the-google-cloud-platform-gcp) or a [local key](./30_Encryption_configuration.md#configure-csfle-with-local-key).
\ No newline at end of file
+In case you want to use [Client Side Field Level Encryption](https://docs.mongodb.com/manual/core/security-client-side-encryption/),
+you should also include several specific Environment Variables, either you are using the [encryption with Google Platform Cloud](./30_Encryption_configuration.md#configure-csfle-with-the-google-cloud-platform-gcp)
+or a [local key](./30_Encryption_configuration.md#configure-csfle-with-local-key).
```

Comment:
```suggestion
In case you want to use [Client Side Field Level Encryption](https://docs.mongodb.com/manual/core/security-client-side-encryption/), you should also include several specific Environment Variables, either you are using the [encryption with Google Platform Cloud](./30_Encryption_configuration.md#configure-csfle-with-the-google-cloud-platform-gcp) or a [local key](./30_Encryption_configuration.md#configure-csfle-with-local-key).
```

Reply:
I decided to split the sentence across multiple lines to easy reading it in the "raw" README file.